warehouse/packaging/models.py

```diff
@@ -296,10 +296,12 @@ def __acl__(self):
             permissions |= {(role.user_id, "Administer") for role in query.all()}
 
         for user_id, permission_name in sorted(permissions, key=lambda x: (x[1], x[0])):
+            permissions = ["upload"]
```

Comment:
just a nit, but this could lead to some trouble if another if or for block or whatever is added after this down the line where we're replacing the value of the `permissions` set that we're looping over.

Reply:
Yeah, good point -- this is a vestigial change from the original ACL approach that we've discarded, so I can drop it entirely.

Reply:
This still needs an update.